src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -50,5 +65,35 @@ public CacheOptions(bool useSharedCache)
         /// </remarks>
         public bool UseSharedCache { get; set; }
 
+        private long? _sizeLimit;
+
+        /// <summary>
+        /// Total token cache size limit in bytes for both app and user token caches.
```

Comment:
Please add a link to the caching doc and also explain that this is not a concern if distributed caching is used.